README.md

```diff
@@ -2,45 +2,162 @@
 
 [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/cpp/cpp%20-%20client%20-%20ci?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1611&branchName=master)
 
-This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp) or our versioned [developer docs](https://azure.github.io/azure-sdk-for-cpp).
+This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp).
 
 ## Getting started
 
-To get started with a library, see the **README.md** file located in the library's project folder. You can find these library folders grouped by service in the `/sdk` directory.
+For the best development experience, we recommend that developers use the [CMake projects in Visual Studio](https://docs.microsoft.com/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) to view and build the source code together with its dependencies. You can also use any other text editor of your choice, such as [VS Code](https://code.visualstudio.com/), along with the command line for building your application with the SDK.
 
-For tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developers](https://azure.github.io/azure-sdk-for-cpp).
+You can find additional information for specific libraries by navigating to the appropriate folder in the `/sdk` directory. See the **README.md** file located in the library's project folder. For example, [here is the doc for the storage client library](https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage#azure-storage-client-library-for-c).
 
-## Packages available
-Each service might have a number of libraries available from each of the following categories:
-* [Client - New Releases](#client-new-releases)
-* [Client - Previous Versions](#client-previous-versions)
+For API reference docs, tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developer Docs](https://azure.github.io/azure-sdk-for-cpp).
+
+### Download & Install the SDK
+
+The easiest way to get the C++ SDK is to leverage [vcpkg](https://github.com/microsoft/vcpkg). You'll need to have [Git](https://git-scm.com/downloads) installed before getting started.
+
+First clone and bootstrap vcpkg itself. You can install it anywhere on your machine, but **make note** of the directory where you clone the vcpkg repo.
+
+```cmd
+> git clone https://github.com/microsoft/vcpkg
+```
+
+On Windows:
+
+```cmd
+> .\vcpkg\bootstrap-vcpkg.bat
+```
+
+On Linux:
+
+```sh
+> ./vcpkg/bootstrap-vcpkg.sh
+```
+
+To install the libraries for your project, run the following, optionally specifying the triplet:

```

Comment:
If we go to explain vcpkg basics, I'd specify, what the triplet is: "the triplet is a combination of a, b, and c [link to vcpkg docs]". And in this sentence, "optionally specifying the triplet (such as X, Y etc.)"

Reply:
I'd avoid too much detail and instead show "Install x64 on Windows" followed by steps.  Have 2 or 3 of the basic exact commands.  Then link to vcpkg for those that want a more in depth explanation.  
We should avoid being documentation for vcpkg.  Also you should wait to provide deep dive links to external sites as each time its an opportunity to "lose" them in the journey.

Reply:
Agreed, that was my approach with the docs here.